content/using/install.md

```diff
@@ -8,35 +8,29 @@ aliases = ["/docs/getting-started/"]
 
-In order to get going on the network, you'll need an [Urbit ID](#id). To try the live network out with a disposable identity, you can always [create a comet](@/using/operations/creating-a-comet.md).
-
 ## Overview
 
 There are two major components of the Urbit ecosystem: **Urbit OS** and **Urbit ID**.
 
-The the Urbit peer-to-peer (p2p) network is composed of instances of Urbit OS -- callled **ships** -- that are virtual machines that run on any Unix platform.
+The the Urbit peer-to-peer (p2p) network is composed of instances of Urbit OS -- called **ships** -- that are virtual machines that run on any Unix platform.
 
-Urbit works as a p2p network where others fail because ships are associated with persistent and scarce identities that we refer to as Urbit IDs. The Urbit ID registry is deployed on the Ethereum blockchain, and individual IDs exist as non-fungible ([ERC-721](https://eips.ethereum.org/EIPS/eip-721)) tokens in that cryptographic ecosystem that chain. While Urbit ID is the essential crytopgraphic foundation to the functioning of the Urbit network, it has uses far beyond that. It can be used as general-purpose identity infrastructure for any other project.
+Urbit works as a p2p network where others fail because ships are associated with persistent and scarce identities that we refer to as Urbit IDs. The Urbit ID registry is deployed on the Ethereum blockchain, and individual IDs exist as non-fungible ([ERC-721](https://eips.ethereum.org/EIPS/eip-721)) tokens.
 
 On the network, Urbit IDs function as network addresses, represented by pronounceable names like `~padmyn-pasnux`. There are a few kinds of Urbit ID, but ones meant for typical users are called **planets**. Anyone who has a planet can be thought of as a full "citizen" of the Urbit network. But to run your ship on the network, you need to start it with a piece of cryptographic information from its Urbit ID. This guide walk you through getting onto Urbit with an ID.
 
-But sometimes a user might not want a persistent identity for whatever reason. In that case, they can use Urbit without an Urbit ID by using something called a **comet**. Comets can be spun up at will and for free. They are good for bots and for people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible peer. If you want to jump right in with a comet, check out [this](#comet) section.
+There are also free IDs meant for temporary use called **comets**. They are good for bots and people who are new to Urbit and want to check out the network, but they aren't good for building a reputation as a friendly and responsible peer. Their long names make them difficult to remember and their low default reputation makes them less likely to get over the air updates. Some communities ban comets to prevent spam, but they're still a good way to see what Urbit is about before buying your own planet.
```

Comment:
Object to saying comets are for temporary use.  Comets have many potential uses completely tangential to a sort of trial mode of urbit.

Reputation is not what prevents them from getting OTAs.

Reply:
What prevents them from getting OTAs? I asked on chat and someone told me it was because they don't get prioritized for compute since they have low default reputation.

Reply:
I'll change the language to remove 'temporary'